distribution/changelog.txt

```diff
@@ -37,6 +37,7 @@
 - Fix: [#19955] Mine Train Roller Coaster has incorrect supports on the sloped left small turn (original bug).
 - Fix: [#19987] [Plugin] ‘SetCheatAction’ has wrong ID in plugin API.
 - Fix: [#19991] Crash using cut-away view.
+- Fix: [#19935] Part of old multiplayer name remains when entering a shorter name.
```

Comment:
This is a regression, so no changelog entry is necessary.

Reply:
[82cbf57](https://github.com/OpenRCT2/OpenRCT2/pull/20004/commits/82cbf57ec61abe1fa39c3c88b40d44a22632766b)